packages/client/wallets/aa/src/blockchain/wallets/EVMAAWallet.ts

```diff
@@ -213,7 +212,7 @@ export class EVMAAWallet extends LoggerWrapper {
         });
     }
 
-    public getSigner<Type extends SignerType>(type: Type): SignerMap[Type] {
```

Comment:
no point in having this elaborate type setup w/ one signer type. Simplifying where we can here